src/stores/temperature.js

```diff
@@ -0,0 +1,7 @@
+import { writable } from 'svelte/store';
```

Comment:
Why do we need store if we save data in local storage? Can it be done like in `searchHistory`, just with util?

Reply:
I see your point. I'll check this suggestions. I've add the store for reactive temperature unit subscription. I use it for translation values and temperature sign in runtime.